workers/unity/Packages/com.improbable.gdk.core/Authority/AuthorityChanges.cs

```diff
@@ -4,6 +4,10 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     This ECS Component stores a ordered list of authority changes received in a single frame.
```

Comment:
This doesn't need to mention that it's an ECS component.

Reply:
I think its important to declare the type of component given the overloading (SpatialOS component, ECS component, GameObject component)